out/artifacts/trivialTrivia_Web_exploded/jsp/form.jsp

```diff
@@ -0,0 +1,70 @@
+<div class="p-3">
+    <form action="#" method="post" id="form-api" class="border p-2">
+        <fieldset>
+            <legend class="text-center">Api Results</legend>
+
+            <div class="form-group">
+                <label for="trivia-amount">Number of Questions:</label>
+                <input type="number" name="triviaAmount" id="trivia-amount" class="form-control" min="1" max="50" value="10">
```

Comment:
I forget, are we only accepting 10 here? I think you mentioned something about that before..?

Reply:
i think its okay not to limit it since this form part is just for demo during the presentation, i think when we mentioned limiting before was when we thought we needed to actually build out an app rather than just the api. The original api itself has a limit of 50 so i just set it to 50 on the form.